### PR TITLE
update script

### PR DESCRIPTION
**Pull request by wileyj**
### Applicable issues

- fixes #

### Checklist

- [ ] Test coverage for new or modified code paths
- [ ] Changelog is updated
- [ ] Required documentation changes (e.g., `docs/rpc/openapi.yaml` and `rpc-endpoints.md` for v2 endpoints, `event-dispatcher.md` for new events)
- [ ] New clarity functions have corresponding PR in `clarity-benchmarking` repo
- [ ] New integration test(s) added to `bitcoin-tests.yml`

### CODE REVIEW

**Comment by wileyj** [comment]
comment

**Comment by stacks-foundation** [comment]
comment1

File: test.sh [hunk 1]
```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
 echo "line 1"
+echo "line 2"
```
Comment (stacks-foundation) [inline test.sh:3]:
```suggestion
echo "line 2"
echo "line 3"
```

**Review by stacks-foundation — CHANGES_REQUESTED** [review]
make changes

**Review by stacks-foundation — DISMISSED** [review]
approve

File: test-2.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+echo "line 1"
+echo "line 2"
```
Comment (stacks-foundation) [inline test-2.sh:3]: this is a line

**Review by stacks-foundation — APPROVED** [review]
fff

**Comment by wileyj** [comment]
This pull request has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**